packages/table/src/cell/formats/truncatedFormat.tsx

```diff
@@ -202,6 +219,32 @@ export class TruncatedFormat extends React.Component<ITruncatedFormatProps, ITru
                 return detectTruncation
                     ? this.state.isTruncated
                     : truncateLength > 0 && content.length > truncateLength;
+            case TruncatedPopoverMode.WHEN_TRUNCATED_APPROX:
+                if (!detectTruncation) {
+                    return truncateLength > 0 && content.length > truncateLength;
+                }
+                const {
+                    getApproximateCharWidth: approxCharWidth,
+                    getApproximateLineHeight: approxLineHeight,
+                    getCellHorizontalPadding: horizontalPadding,
+                    getNumBufferLines: numBufferLines,
```

Comment:
Do you plan to support functions for these properties too? The names are misleading otherwise. (Granted, if there's no public API, it doesn't matter, but it probably deserves a comment then.)

Reply:
This is what I get for copy/pasting :P 

We should probably support a public API for this, but given that this is on the cell level they probably don't need to be functions. I'll change the names and make them public, I think.